### ISSUE TITLE
HTML File exported is not loadable in browser

### ISSUE THREAD

**Issue by SuschNL** [KosmosisDire/obsidian-webpage-export#228]
**Describe the bug**
Export of HTML file creates a file that displays nothing in the browser. Tried both Chrome and Edge. Note that I used to be able to just select "Export as HTML" from the three-dot menu on the upper right of the Obsidian document to export the HTML. Now I seem to get some tree list of all my documents and I have to hunt through it for the document I am actually editing to get that to export. Can I get the old interface back?

**Operating System**
Windows 10

**To Reproduce**
1. Select "Export as HTML"
2. Find document in tree list and select
3. Click Export
4. *Note, file is not opened after export, even though that is selected*
5. Find document in path - note that document is now buried in tree-folder structure that wasn't there in previous version
6. Open document in browser
7. Get blank screen

### IMPORTANT: Error Log
This is what I see if I go into developer mode in Chrome:

```
Uncaught (in promise) TypeError: Cannot set properties of null (setting 'checked')
    at e (Database Schema Change 2023-09-13.html:1:192702)
    at setupThemeToggle (Database Schema Change 2023-09-13.html:1:193465)
    at initializePage (Database Schema Change 2023-09-13.html:1:159386)
    at window.onload (Database Schema Change 2023-09-13.html:1:161227)
```

**Comment by KosmosisDire** [comment]
Hi, you can still use the corner menu to export your files like before. Is the "Export as HTML" button not showing?

![image](https://github.com/KosmosisDire/obsidian-webpage-export/assets/39423700/646d666b-3019-4a28-94bd-99117775b67a)

Here you can see it is still there and clicking it will open the popup with only that file selected in the tree.

Is it possible for you to send me the broken .html file? And you can you also look at the developer log in obsidian, not just chrome

Thanks!

**Comment by SuschNL** [comment]
Issue seems to be the "theme" toggle. Output works correctly if I turn on "Include theme toggle" in the settings. I normally have that off.

BTW, I used that "Export as HTML" but the current file is not selected. I had to go find it.

**Comment by KosmosisDire** [comment]
Ahh okay, interesting I am able to reproduce this I will look into a fix. Can you make a new issue for the file picker not checking the exported file? Thanks! :)

**Comment by SuschNL** [comment]
I can't seem to recreate the file picker issue. Maybe it was me? I'll let you know if I see any issues with it in the future.

Thank you!

**Comment by KosmosisDire** [comment]
Okay cool! Thank you! :)

**Comment by KosmosisDire** [comment]
I am aware of the issue and it will be fixed in the next release, thanks :)

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
